packages/client/mutations/SendClientSegmentEventMutation.ts

```diff
@@ -9,7 +9,7 @@ const mutation = graphql`
 `
 
 interface Options {
-  eventId: number
+  eventId?: number
```

Comment:
`eventId` is being used in [useSegmentTrack](https://github.com/ParabolInc/parabol/blob/feadf80b9ab8561b495c5e9a43cddd3e8e07a918/packages/client/hooks/useSegmentTrack.ts) to check whether the hook is firing too frequently. It's not a requirement for Segment so I think it can be optional

Reply:
oh yeah! i'll go ahead & remove that, that issue was resolved